framework/core/src/Extension/Console/ToggleExtensionCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extension\Console;
+
+use Flarum\Console\AbstractCommand;
+use Flarum\Extension\ExtensionManager;
+
+class ToggleExtensionCommand extends AbstractCommand
+{
+    /**
+     * @var ExtensionManager
+     */
+    protected $extensionManager;
+
+    public function __construct(ExtensionManager $extensionManager)
+    {
+        $this->extensionManager = $extensionManager;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('extension:toggle')
+            ->setDescription('Enable or disable an extension.')
+            ->addArgument('extension', null, 'The extension to enable or disable.')
+            ->addOption('enable', null, null, 'Enable the extension.')
+            ->addOption('disable', null, null, 'Disable the extension.');
```

Comment:
rather than `extension:toggle --enable` or so, I'd prefer `extension:enable` and `extension:disable`. I'd also like us to be specific about the argument's nature, which is the extension ID.
```suggestion
        $this
            ->setName('extension:enable')
            ->setAliases(['extension:disable'])
            ->setDescription('Enable or disable an extension.')
            ->addArgument('extension-id', null, 'The ID of the extension to enable or disable.');
```